contrib/babelfishpg_tsql/src/hooks.c

```diff
@@ -3477,6 +3477,50 @@ pltsql_store_func_default_positions(ObjectAddress address, List *parameters, con
 
 	if (HeapTupleIsValid(oldtup))
 	{
+		StringInfoData infoSchemaStr;
+
+		initStringInfo(&infoSchemaStr);
+
+		for(int i = 0; i < strlen(original_query); i++)
+		{
+			if(original_query[i] == '-')
+			{
+				while(original_query[i] != '\n' && i < strlen(original_query))
+				{
+					appendStringInfoChar(&infoSchemaStr, original_query[i]);
+					i++;
+				}
+				appendStringInfoChar(&infoSchemaStr, original_query[i]);
+			}
+			else if(original_query[i] == '/' && original_query[i + 1] == '*' && i < strlen(original_query))
+			{
+				while(original_query[i] != '*' && original_query[i+1] != '/')
+				{
+					appendStringInfoChar(&infoSchemaStr, original_query[i]);
+					i++;
+				}
+				appendStringInfoChar(&infoSchemaStr, original_query[i]);
+				appendStringInfoChar(&infoSchemaStr, original_query[i+1]);
+				i++;
+			}
+			else if(strncasecmp(original_query + i, "create", 6) == 0)
```

Comment:
Will it be wrong is original_query pointer has less then 6 char left before it's end ?